layers/subresource_adapter.cpp

```diff
@@ -282,7 +282,7 @@ ImageRangeEncoder::ImageRangeEncoder(const IMAGE_STATE& image, const AspectParam
     VkImageSubresourceLayers subres_layers = {limits_.aspectMask, 0, 0, limits_.arrayLayer};
     linear_image_ = false;
 
-    // WORKAROUND for dev_sim and mock_icd not containing valid VkSubresourceLayout yet. Treat it as optimal image.
+    // WORKAROUND for dev_sim/profile and mock_icd not containing valid VkSubresourceLayout yet. Treat it as optimal image.
```

Comment:
Looking at the issue it says "all references to devsim should be removed from the validation layers repository."

Is there a reason to leave in the comment about dev_sim?

```suggestion
    // WORKAROUND for profile and mock_icd not containing valid VkSubresourceLayout yet. Treat it as optimal image.
```

Reply:
thinking about it more I guess not, will remove them all where I did this `/profile` thing